sigal/plugins/compress_assets.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+
+"""Plugin to compress static files for faster HTTP transfer.
+
+Currently, 3 methods are supported:
+    - gzip. No dependency required. This is the fastest, but also largest output.
+    - zopfli. Need zopfli module from https://pypi.python.org/pypi/zopfli. gzip compatible output with optimized size.
+    - brotli. Need brotli module from https://pypi.python.org/pypi/Brotli. Brotli is the best compressor for web usage.
+
+"""
+
+from __future__ import unicode_literals
```

Comment:
If you can avoid `unicode_literals`, it's often a source of hidden bugs and I prefer when proper encoding/decoding is done when needed (and I think you don't really need this here).

Reply:
I did this to ensure a proper compatible behavior with python3. Otherwise, the migration is more painful.
However, I think I needed this because of path names that can contains non-ASCII characters. I'll have to check this fact, though.